app/models/schedule.rb

```diff
@@ -5,8 +5,8 @@ class Schedule
 
   belongs_to :sequence
   validates_presence_of :sequence_id
-  belongs_to :user
-  validates_presence_of :user_id
+  belongs_to :device
+  validates_presence_of :device_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.